src/main/java/array/ArrayExercise5.java

```diff
@@ -0,0 +1,20 @@
+package array;
+
+public class ArrayExercise5 {
+    private boolean checkPrime(int a) {
+        for (int i = 2; i <= Math.sqrt(a); i++) {
+            if (a % i == 0)
+                return false;
+        }
+        return true;
+    }
+
+    public int sumOfPrimeNumbers(int[] b) {
+        int sum = 0;
+        for (int j = 0; j < b.length; j++) {
+            if (checkPrime(b[j]) && b[j] != 1)
```

Comment:
```suggestion
            if (checkPrime(b[j]))
```

Em có thêm điều kiện `&& b[j] != 1` bởi vì cái hàm `checkPrime` không hoàn thành đúng nhiệm vụ khi nó trả về `true` khi `a` là `1`. Như vậy nên fix từ hàm `checkPrime` bằng cách

```java
return a >= 2
```

là tốt hơn

Reply:
Done